### PR TITLE
Archive repository

### PR DESCRIPTION
**Pull request by svrnm**
Contributes to https://github.com/open-telemetry/community/issues/2453

I would need someone to translate the text to Japanese as well, such that the note is available in both languages

### CODE REVIEW

**Review by katzchang — APPROVED** [review]

**Review by ymotongpoo — APPROVED** [review]
LGTM

**Comment by svrnm** [comment]
@katzchang @ymotongpoo can one of you provide me with the Japanese translation of the message so we can put it in the banner as well? Thx

**Comment by ymotongpoo** [comment]
@svrnm how about the following?

> このレポジトリはアーカイブされたので、新規のIssueやPull Requestは受け付けていません。
> OpenTelemetryのドキュメントとウェブサイトのローカライズは[ウェブサイトのレポジトリ](https://github.com/open-telemetry/opentelemetry.io)で行っています。

@katzchang could you double check?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # OpenTelemetry ドキュメント日本語化プロジェクト
 
+> [!IMPORTANT]
+> This repository is archived and will not receive any updates or accept issues or pull requests
+>
+> Localizations for the OpenTelemetry documentation and website are now part of the [website repository](https://github.com/open-telemetry/opentelemetry.io)
```
Comment (svrnm) [inline README.md]:
```suggestion
> [!IMPORTANT]
> このレポジトリはアーカイブされたので、新規のIssueやPull Requestは受け付けていません。
> OpenTelemetryのドキュメントとウェブサイトのローカライズは[ウェブサイトのレポジトリ](https://github.com/open-telemetry/opentelemetry.io)で行っています。
>
> This repository is archived and will not receive any updates or accept issues or pull requests
>
> Localizations for the OpenTelemetry documentation and website are now part of the [website repository](https://github.com/open-telemetry/opentelemetry.io)
```

**Comment by svrnm** [comment]
@katzchang @ymotongpoo if you think this is correct please merge the PR, although I have the permissions I would prefer you as maintainers doing it.

**Comment by ymotongpoo** [comment]
I am not sure if I have the permission to merge; as far as I confirmed on GitHub Android app, I don't see the merge button. Also it says still this PR isn't ready for merge, though I am not sure what is required for it. I asked @shirou to review this just in case.

**Review by katzchang — APPROVED** [review]

**Review by shirou — APPROVED** [review]

**Comment by svrnm** [comment]
With all the approvals I went forward and merged it, thanks @shirou, @ymotongpoo and @katzchang -- looking forward to work with you on the docs translation over at the otel website repo :-)

**Pull request merged**